mempool/clist_mempool.go

```diff
@@ -223,6 +218,11 @@ func (mem *CListMempool) Flush() {
 	mem.removeAllTxs()
 }
 
+func (mem *CListMempool) Contains(txKey types.TxKey) bool {
```

Comment:
In a lot of the methods we have a comment before the method such as 
```
// Safe for concurrent use by multiple goroutines.
```
should we include something for this method ?